### PR TITLE
feat(Weblate): create script to automatically merge Weblate changes

### PR DESCRIPTION
**Pull request by sudden6**
see #4981 for how the output looks like

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/4984)

### CODE REVIEW

File: tools/update-weblate.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+#    Copyright © 2016 Zetok Zalbavar <zetok@openmailbox.org>
```
Comment (Diadlo) [inline tools/update-weblate.sh:3]: Update copyright

File: tools/update-weblate.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    
+    git checkout upstream/master
+    
+    local COMMIT_MESSAGE=`git shortlog upstream/master..weblate/master`
+    
+    # squash and merge    
+    git merge --squash --no-edit weblate/master
+    
+    # update format for adapt to Qt translation format
+    ./tools/deweblate-translation-file.sh ./translations/*.ts || true
```
Comment (Diadlo) [inline tools/update-weblate.sh:60]: What if it will be called not from the root or repo?

File: tools/update-weblate.sh (hunk 1 above)
Comment (sudden6) [reply tools/update-weblate.sh:3]: Done

File: tools/update-weblate.sh (hunk 2 above)
Comment (sudden6) [reply tools/update-weblate.sh:60]: It's one of the preconditions needed and stated in the header of the script.

**Comment by sudden6** [comment]
@Diadlo ping?

**Comment by Diadlo** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/qtox/qtox/4984#-:-L8UD96KBThIB_k1YEV4:b-jbsf8z)*

**Pull request merged**